crates/biome_css_formatter/tests/specs/css/declaration_list.css.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: crates/biome_formatter_test/src/snapshot_builder.rs
+assertion_line: 212
```

Comment:
If you see this line,it means you're using an **old version of `cargo-insta`**. Run `just upgrade-tools` to update the tools we use internally.